### DISCUSSION TITLE
How can I change the colour of #tags please?

### DISCUSSION THREAD

**Discussion by BerylTP** [AnubisNekhet/AnuPpuccin#220] [Snippets]
Hello. I'm a newbie here. I love Obsidian and AnuPpuccin. I'm using Luminescence style and would love to know how to change the colour of the #tag.  One colour will do, I just need it to stand out more.   Please could someone tell me like I'm 5 years old how to do it. Thanks in advance.

**Comment by trainbuffer** [comment]
If you start here https://discord.com/channels/686053708261228577/702656734631821413/1106860504766484520 and read down through FireIsGood's post you'll get a better idea about tags. There are posts on the Obsidian forum as well.

Changing the color of all tags is easy. Changing the color of individual tags for only Reading View isn't so bad. Changing the color of individual tags both for Live Preview and Reading views is a chore, but also doable

**Reply by BerylTP** [reply]
Thanks trainbuffer, my #tags are fab now. 🌞
